start.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+function getSandboxPid(){
+    ss -lptn 'sport = :7575' | grep -P -o '(?<=pid=)([0-9]+)'
+}
+function cleanup(){
+    sandboxPID=$(ss -lptn 'sport = :7575' | grep -P -o '(?<=pid=)([0-9]+)')
+    if [[ $sandboxPID ]]; then
+        # kill the sandbox which is running in the background
+        kill $sandboxPID
+    fi
+}
+
+trap cleanup ERR EXIT
+
+daml build
+packageId=$(daml damlc inspect-dar --json .daml/dist/ex-java-bindings-0.0.2.dar | jq '.main_package_id' -r)
+
+echo "Compiling code"
+mvn compile
+
+# Could also run this manually in another terminal without the redirects
+echo "Starting sandbox"
+daml start --start-navigator false --sandbox-port 7600 > sandbox.log 2>&1 & PID=$!
+
+
+while [[ "$(getSandboxPid)" -eq '' ]]
```

Comment:
I recommend that you wait for the port file to be written out not for the port to be open. the latter can be somewhat unreliable.

Reply:
I think this should be fine for now as this is not something super critical and so far it worked out for me fine :)